### ISSUE TITLE
Code quality analysis (scrutinizer like alternative)

### ISSUE THREAD

**Issue by viniarck** [kytos-ng/kytos#191]
- Evaluate code quality analysis scrutinizer like; alternative https://sonarcloud.io/
- It must be compatible with Python 3.9, scrutinizer was lagging behind
- It must integrate well with GitHub actions

**Comment by viniarck** [comment]
Also include in this scope, whether this solution can also report code coverage results, or open a new issue to support codecov.io or coveralls

**Comment by viniarck** [comment]
I've removed the 2022.3 label since scrutinizer has been working well and is correctly supporting Py3.9 version. We don't have the capability to run multiple inspections in parallel but that doesn't impact much at the moment since it runs relatively fast and we don't have hundreds of PRs

**Comment by viniarck** [comment]
Closing this since so far scrutinizer is meeting the expectations. This issue might get reopened if that changes in the future.
